Tools/Scripts/webkitdirs.pm

```diff
@@ -2805,7 +2806,10 @@ sub generateBuildSystemFromCMakeProject
 
     push @args, "-DLTO_MODE=$ltoMode" if ltoMode();
 
-    if (isPlayStation()) {
+    if (isAnyWindows()) {
```

Comment:
As above, this seems like a bad choice of condition as it appears to be based on what OS perl was built for. We presumably want to do this for the windows port or jsconly on windows but not for another port that happens to be being built on windows.